project_event/views/project_task_view.xml

```diff
@@ -188,6 +183,7 @@
         <field name="domain">[('activity_task_type','=','activity')]</field>
         <field name="context" eval="{'default_activity_task_type': 'activity'}"/>
         <field name="view_mode">tree,form,calendar</field>
+        <field name="search_view_id" ref="project_activity_search"/>
```

Comment:
I'm getting an error here:
**ParseError**: "External ID not found in the system: project_event.project_activity_search" while parsing /opt/odoodev/git/project-addons/project_event/views/project_task_view.xml:
__179__